provider/oci/provider.go

```diff
@@ -41,27 +41,6 @@ type environConfig struct {
 var _ config.ConfigSchemaSource = (*EnvironProvider)(nil)
 var _ environs.ProviderSchema = (*EnvironProvider)(nil)
 
-// var cloudSchema = &jsonschema.Schema{
-// 	Type:     []jsonschema.Type{jsonschema.ObjectType},
-// 	Required: []string{cloud.RegionsKey, cloud.AuthTypesKey},
-// 	Order:    []string{cloud.RegionsKey, cloud.AuthTypesKey},
-// 	Properties: map[string]*jsonschema.Schema{
-// 		cloud.RegionsKey: {
-// 			Type:     []jsonschema.Type{jsonschema.ObjectType},
-// 			Singular: "region",
-// 			Plural:   "regions",
-// 			AdditionalProperties: &jsonschema.Schema{
-// 				Type: []jsonschema.Type{jsonschema.ObjectType},
-// 			},
-// 		},
-// 		cloud.AuthTypesKey: {
-// 			// don't need a prompt, since there's only one choice.
-// 			Type: []jsonschema.Type{jsonschema.ArrayType},
-// 			Enum: []interface{}{[]string{string(cloud.HTTPSigAuthType)}},
-// 		},
-// 	},
-// }
```

Comment:
Delete me.

Reply:
these lines are deleted on this PR actually

Reply:
Ah yeah, whoops.